ehsm_kms_service/quote_manager_apis.js

```diff
@@ -0,0 +1,205 @@
+const logger = require('./logger')
+const { v4: uuidv4 } = require('uuid')
+const {
+    QUOTE_LENGTH_MAX,
+    NONCE_LENGTH_MAX,
+    CHALLENGE_LENGTH_MAX
+} = require('./constant')
+const {
+    napi_result,
+    _result,
+    base64_encode,
+    base64_decode,
+    gen_hmac
+} = require('./function')
+const {
+    cryptographic_apis
+} = require('./apis')
+
+
+/**
+ * Verify that the parameter is string and whether it is required and the maximum length
+ * @param {object} param : Variables requiring validation
+ * @param {boolean} required : Whether the parameter is required
+ * @param {int} maxLength [defalut=undefined] : Maximum length of string, If maxLength is undefined, the length is not verified.
+ * @returns {boolean}
+ */
+function checkStringParam(param, required, maxLength) {
```

Comment:
can you update the checking logic in this patch?

Reply:
will fix it on next patch